data/json/itemgroups/magazines.json

```diff
@@ -169,7 +169,9 @@
       ["falbigmag", 50],
       ["g3mag", 80],
       ["g3bigmag", 40],
+      ["m107a1mag", 50],
```

Comment:
I'm thinking these two magazines should be in `mags_milspec`. It seems inappropriate for the aftermarket `ruger1022bigmag` to have equivalent frequency to this